static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
         &:before {
             .cf-icon();
-            content: "\e402";
+            content: "\e406";
```

Comment:
Opened https://github.com/cfpb/cfgov-refresh/issues/354 for this, since this would be good to do for all unicode references in the CSS.